tedge/src/cli/connect/command.rs

```diff
@@ -404,7 +404,13 @@ fn new_bridge(
     config_location: &TEdgeConfigLocation,
 ) -> Result<(), ConnectError> {
     println!("Checking if {} is available.\n", service_manager.name());
-    let () = service_manager.check_operational()?;
+    let service_manager_result = service_manager.check_operational();
+
+    if let Err(SystemServiceError::ServiceManagerUnavailable(name)) = &service_manager_result {
+        println!("It appears that '{}' service manager is installed on the system, thin-edge.io works seamlessly with 'systemd'.", name);
```

Comment:
```suggestion
        println!("It appears that '{}' service manager is not installed on the system, thin-edge.io works seamlessly with 'systemd, bsd and openrc based systems'.", name);
```

Reply:
Good catch.

I don't want to mention other managers here as we are officially supporting systemd and this was the request to advertise better experience with systemd.